plugins/catalog/src/components/CatalogTable/CursorPaginatedCatalogTable.tsx

```diff
@@ -30,12 +30,10 @@ type PaginatedCatalogTableProps = {
  */
 
 export function CursorPaginatedCatalogTable(props: PaginatedCatalogTableProps) {
-  const { columns, data, next, prev, title, isLoading, options, ...restProps } =
-    props;
+  const { columns, data, next, prev, options, ...restProps } = props;
 
   return (
     <Table
-      title={isLoading ? '' : title}
```

Comment:
I can't think of the advantage of hiding the title while the catalog is loading, but if anyone wants to convince me I'll make an equivalent change to the other Tables

Reply:
I would like to double-check with @sennyeya if he can provide context on the change as it is added to this pull request 🙂 :
https://github.com/backstage/backstage/pull/22527/files#diff-9ddb0bc844e0ae3d0c342e44fdd890a4fe3a9c60ff119a4a7be58b25b3b7f9f7R40

Reply:
It was a while ago, but I want to say the API response was zero-ing out the title, so I removed it while it was loading. If that's no longer the case, this can be removed.

Reply:
I don't remember seeing this behaviour, but again I'm happy to keep this behaviour, I'd just do the same thing for all the different implementations

Reply:
We can add it back if it's an issue.